apps/spruce/src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -145,6 +145,7 @@ export const TaskDurationTable: React.FC<Props> = ({
         header: "Build Variant",
         size: 150,
         enableColumnFilter: true,
+        enableSorting: true,
```

Comment:
What's the reason for adding `enableSorting` to these columns? Other tables like the TestsTable just set `enableMultisort` at the top level of the table since all columns are usable in multisort

Reply:
I think this is required. I tried removing the `enableSorting` from each column and the sorts don't show up. TestsTable applies this value to the columns. Check it out [here](https://github.com/evergreen-ci/ui/blob/7a69ecadec70775e06d1dd8b01793117ad0824ff/apps/spruce/src/pages/task/taskTabs/testsTable/getColumnsTemplate.tsx#L36)